payone-adapter/src/main/java/com/commercetools/payment/payone/config/PayoneConfigurationNames.java

```diff
@@ -20,4 +20,16 @@
     static final String PAID_TO_IBAN = "paidToIBAN";
     static final String PAID_TO_BIC = "paidToBIC";
     static final String PAID_TO_NAME = "paidToAccountHolderName";
+
+    // KLARNA specific options
```

Comment:
actually, the fields are not only Klarna specific, but for Klarna the have different mandatory rule.

Update the comments: 5c41b78